### ISSUE TITLE
Request to add SmoothFlowLayoutManager

### ISSUE THREAD

**Issue by tcqq** [davideas/FlexibleAdapter#654]

**Comment by davideas** [comment]
@tcqq, which effect exactly? The multi-elements with different column size and rows? Isn't a capability of the LayoutManager you adopt (not the adapter)?

**Comment by tcqq** [comment]
@davideas Yes, this is a FlowLayout. It is a LayoutManager. Using Google's flexbox-layout will achieve this effect, but Google's inventory is in issues.

**Comment by tcqq** [comment]
@davideas If you can fixed the [#453](https://github.com/google/flexbox-layout/issues/453) issue in flexbox-layout or fixed the [#1](https://github.com/Astrocode011235813/FlowLayoutManager/issues/1) and [#2](https://github.com/Astrocode011235813/FlowLayoutManager/issues/2) issues in FlowLayoutManager, FlexibleAdapter will be able to implement FlowLayout effect very quickly.
